hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpc/AsyncServerSocketTest.java

```diff
@@ -151,49 +152,46 @@ public void test_bind_whenAlreadyBound() {
         socket.close();
     }
 
-//    @Test
-//    public void test_accept_whenConsumerNull() {
-//        Reactor reactor = newReactor();
-//        AsyncServerSocket socket = reactor.newAsyncServerSocketBuilder().build();
-//
-//        SocketAddress local = new InetSocketAddress("127.0.0.1", 5000);
-//        socket.bind(local);
-//
-//        assertThrows(NullPointerException.class, () -> socket.start(null));
-//
-//        socket.close();
-//    }
-//
-//    @Test
-//    public void test_createCloseLoop_withSamereactor() {
-//        SocketAddress local = new InetSocketAddress("127.0.0.1", 5000);
-//        Reactor reactor = newReactor();
-//        for (int k = 0; k < 1000; k++) {
-//            System.out.println("at:" + k);
-//            AsyncServerSocket socket = reactor.newAsyncServerSocketBuilder()
-//                    .set(AsyncSocketOptions.SO_REUSEPORT,true)
-//                    .build();
-//            socket.bind(local);
-//            socket.start(socket1 -> {
-//            });
-//            socket.close();
-//        }
-//    }
-//
-//    @Test
-//    public void test_createCloseLoop_withNewreactor() {
-//        SocketAddress local = new InetSocketAddress("127.0.0.1", 5000);
-//        for (int k = 0; k < 1000; k++) {
-//            System.out.println("at:" + k);
-//            Reactor reactor = newReactor();
-//            reactors.remove(reactor);
-//            AsyncServerSocket socket = reactor.newAsyncServerSocketBuilder()
-//                    .set(AsyncSocketOptions.SO_REUSEPORT,true)
-//                    .build();
-//            socket.bind(local);
-//            socket.start(socket1 -> {
-//            });
-//            terminate(reactor);
-//        }
-//    }
+    @Test
+    public void test_createCloseLoop_withSamereactor() {
+        SocketAddress local = new InetSocketAddress("127.0.0.1", 5000);
+        Reactor reactor = newReactor();
+        for (int k = 0; k < 1000; k++) {
+            System.out.println("at:" + k);
```

Comment:
Minor: Should we keep these prints? (this and the one in line 180)

Reply:
I already removed them. Debugging leftovers.